project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_users/c4m_og_invite_users.module

```diff
@@ -108,3 +108,92 @@ function c4m_og_invite_users_mail_tokens(array &$replacements, array $data, $opt
   $replacements['[group:accept_invite_url]'] = $answer_link;
   $replacements['[site:name]'] = variable_get('site_name', 'Capacity 4 More');
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ *
+ * Add a subject field to the invitation form.
+ */
+function c4m_og_invite_users_form__og_invite_form_alter(&$form, &$form_state, $form_id) {
+  $form['subject'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Subject'),
+    '#weight' => 10,
+  );
+
+  $form['message']['#weight'] = 20;
+  $form['#submit'][0] = 'c4m_og_invite_users_form_submit';
+}
+
+/**
+ * Invitation form submit callback.
+ */
+function c4m_og_invite_users_form_submit($form, &$form_state) {
+  $group_type = $form_state['values']['group_type'];
+  $gid = $form_state['values']['gid'];
+  $message = check_plain($form_state['values']['message']);
+  $subject = check_plain($form_state['values']['subject']);
+  $invitees = explode(',', str_replace(', ', ',', $form_state['values']['invitee']));
+
+  $entity = entity_load_single($group_type, $gid);
+  $label = entity_label($group_type, $entity);
+  $field_name = key(og_get_group_audience_fields());
+  foreach ($invitees as $invitee) {
+    if (empty($invitee)) {
+      continue;
+    }
+    list($name, $uid) = explode('||', preg_replace('#^(.*)\s\[([0-9]+)\]$#', '$1||$2', $invitee));
+
+    $result = c4m_og_invite_users_invite($group_type, $gid, $field_name, $uid, $message, $subject);
+
+    if (!empty($result)) {
+      drupal_set_message(t('<em>@user</em> has been invited to the group <em>@label</em>.', array('@user' => $name, '@label' => $label)));
+      $form_state['redirect'] = 'group/' . $group_type . '/' . $gid . '/admin/people/invite-users';
+    }
+    else {
+      drupal_set_message(t('An error happened while inviting <em>@user</em> to the group <em>@label</em>', array('@user' => $name, '@label' => $label)), 'error');
+    }
+  }
+}
+
+/**
+ * Send an invitation to an user.
+ */
+function c4m_og_invite_users_invite($group_type, $gid, $field_name, $uid, $message = '', $subject = '', $inviter = NULL) {
+  if (empty($inviter)) {
+    global $user;
+    $inviter = $user;
+  }
+  $invitee = user_load($uid);
+  $group = entity_load_single($group_type, $gid);
+  $subject = $subject ?: 'Invitation to join the ' . $group->type . ' ' . $group->title;
```

Comment:
Please use the regular `if` so it will be more readable, plus, wrap this text with a `t()` function.